### PR TITLE
Fixes point of sale description

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Turned point of sale into a phone subtype.
Fixes point of sale description.
## Why It's Good For The Game

Broken descriptions are bad.
Having it become a proper item avoids repetition and makes it easier to be used in other places.
## Images of changes

BEFORE
pointofsaled

AFTER
after

## Testing

- Verified the item description, all looks good.
## Changelog
:cl:
spellcheck: Point of sale description has been fixed
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/misc.dm [hunk 1]
```diff
@@ -145,3 +145,10 @@
 	if(cooldown < world.time - 20)
 		playsound(user.loc, 'sound/weapons/ring.ogg', 50, 1)
 		cooldown = world.time
+
+/obj/item/phone/pos
+	name = "point of sale"
+	desc = "Also known as a cash register, or, more commonly, \"robbery magnet\". It's old and rusty, clearly non-functional and decorative only."
+	icon = 'icons/obj/machines/pos.dmi'
+	icon_state = "pos"
+	attack_verb = list("bounced a check off","checked-out","tipped")
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/misc.dm:154]:
```suggestion
	attack_verb = list("bounced a check off", "checked-out", "tipped")
```

**Review by Sirryan2002 — APPROVED** [review]
code looks good, mapping changes should be good aswell

**Comment by S34NW** [comment]
I hate these sprites

**Review by SteelSlayer — APPROVED** [review]
That var-editing *is* disgusting, wow

**Pull request merged**